greenpithumb/camera.py

```diff
@@ -0,0 +1,27 @@
+class Camera(object):
```

Comment:
I'd say it's more like a `CameraManager` or a `PhotoSaver` because the only thing the caller can do with it is tell it to save a picture to the filesystem.